edgelet/edgelet-http/Cargo.toml

```diff
@@ -12,6 +12,8 @@ failure_derive = "0.1"
 futures = "0.1"
 http = "0.1"
 hyper = { version = "0.11", features = ["compat"] }
+hyper-proxy = "0.4.1"
+hyper-tls = "0.1"
```

Comment:
Bump for my 3rd party OSS updates comment.

Reply:
Done.